operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -87,20 +87,20 @@ spec:
     #     description: db-sync pod not running
 
     - alert: MessagebusNotRunning
-      expr: up{job="messagebus"} < 1
+      expr: sum(up{job="messagebus"}) by (cluster) < 1
```

Comment:
We use `sum` now because with the centralization, the metric can be aggregated across all Webapp clusters?

Reply:
Yep, with the centralization there would be 2 things happening:
1. There would be multiple series for the raw metric - 1 for each cluster
2. Aggregations can only be applied to summary metrics, so we need to sum anyway to be able to group by cluster